calicoctl/commands/node/status.go

```diff
@@ -160,5 +193,11 @@ func drawTable(birdOut string) {
 	for _, v := range data {
 		table.Append(v)
 	}
+
+	if len(data) == 0 {
+		fmt.Printf("No IPv%s address configured.\n", ipv)
```

Comment:
We don't have to print the _exact_ same message as the previous calicoctl version :)

It's generally better to make the message describe what is actually happening, otherwise it could be confusing to users.

In this case, we print this when no IPvX peers were found, I think.